opsdroid/core.py

```diff
@@ -488,10 +488,10 @@ def _connector_names(self):  # noqa: D401
         """
         self.stats["messages_parsed"] = self.stats["messages_parsed"] + 1
         tasks = []
+        tasks.append(self.eventloop.create_task(parse_event_type(self, event)))
```

Comment:
Good spot with this. That was my mistake.

The `tasks.append(self.eventloop.create_task(parse_always(self, event)))` line should probably be up here too.

Reply:
Done.